.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# CODEOWNERS file for das-download-service.
+# The following users will be automatically added as Reviewers to all PRs.
+
+* * @MarkFCain @MAli99 @DevenKShah @raviudari01
```

Comment:
These is an extra * here. GitHub does not like it :-)

Reply:
DOH! Fixed